src/network/networkd-link.c

```diff
@@ -179,6 +179,12 @@ static void link_update_master_operstate(Link *link, NetDev *netdev) {
         if (netdev->ifindex <= 0)
                 return;
 
+        /* If the interface is self-mentioned in Bridge= or friends, then this introdduce infinite loop.
```

Comment:
Typo here. But since this will need to be reworked for the FIXME, let's fix that then.